ROADMAP.md

```diff
@@ -0,0 +1,33 @@
+# Kubeflow Distributed Training Operators 2020 Roadmap
+
+This document outlines the main directions on the Kubeflow Distributed Training Operators in 2020.
+
+## Maintenance and reliability
+
+We will continue developing capabilities for better reliability, scaling, and maintenance of production distributed training experiences provided by operators.
+
+* Enhance maintainability of operator common module https://github.com/kubeflow/common/issues/54
+* Migrate operators to use kubeflow/common apis
```

Comment:
Do you mean tf and pytorch?

Reply:
TF and Pytorch is already on it. PyTorch still import TF libraries. I think mxnet is another one.

Reply:
Gotcha.

Reply:
But, TF and Pytorch is not using common implementation yet.

Reply:
Tree based model like XGBoost or LightGBM would use this. XGBoost operator is implemented via Common.